app/slices/Slice.scala

```diff
@@ -942,25 +942,25 @@ case object ShowcaseSingleStories extends Slice {
  * The implementation on platforms effectively limits the display to 6 cards altogether, and only 2-4 cards at a time.
  * In the tool, we're satisfied with using a 6 card layout to hint at the number of visible stories.
  *
- * Desktop:
- * .____________.____________.____________.____________.____________.____________.
- * |       #####|       #####|       #####|       #####|       #####|       #####|
- * |       #####|       #####|       #####|       #####|       #####|       #####|
- * |       #####|       #####|       #####|       #####|       #####|       #####|
- * '-----------------------------------------------------------------------------'
+ * Desktop (three cards + peeking card):
```

Comment:
Nice tweaks on the ASCII art, this feels a bit more representative! (And vaguely makes me wonder how it gets turned into the actual images on the tool, since the images seem to be missing the helpful `...` bit)